libraries/standard/http/src/private/http_client_parse.h

```diff
@@ -0,0 +1,72 @@
+#ifndef HTTP_CLIENT_PARSE_H_
+#define HTTP_CLIENT_PARSE_H_
+
+#include "http_client.h"
+/* #include "http_parser.h" */
+
+/**
+ * @brief The state of the response message parsed after
```

Comment:
It's the state of the parser, not the response message...
I suggest "States associated with parsing an HTTP response."

Reply:
It is the state of what has been parsed, so it is the state of the response message.